lib/WP_Auth0_Options.php

```diff
@@ -252,4 +247,15 @@ protected function defaults() {
 			'chart_age_step' => '5',
 		);
 	}
+
+	/**
+	 * @deprecated 3.6.0 - Social connections are no longer set during initial setup so this data is no longer needed. 
+	 *
+	 * @return array
+	 */
+	public function get_enabled_connections() {
+		// phpcs:ignore
+		trigger_error( sprintf( __( 'Method %s is deprecated.', 'wp-auth0' ), __METHOD__ ), E_USER_DEPRECATED );
+		return array( 'facebook', 'twitter', 'google-oauth2' );
```

Comment:
is weird that this was (and is) hardcoding an array of enabled connections. I've read the deprecation message. However, are you currently calling this method somewhere in your plugin?

Reply:
I didn't remove it so I'm not entirely sure when or where this was used. I checked through all the functions here and did not see it used anywhere in the code. I think this was part of the initial setup at some point? 

This is the case with a number of this deprecations. I didn't remove them and didn't see them used anywhere. If we're not using the method internally, I don't want to maintain it 👍